jetty-util/src/main/java/org/eclipse/jetty/util/DateCache.java

```diff
@@ -245,17 +242,14 @@ protected Tick formatTick(long now)
     {
         long seconds = now / 1000;
 
-        // Synchronize to protect _tzFormat
-        synchronized (this)
+        // recheck the tick, to save multiple formats
+        if (_tick==null || _tick._seconds!=seconds)
         {
-            // recheck the tick, to save multiple formats
-            if (_tick==null || _tick._seconds!=seconds)
-            {
-                String s= _tzFormat.format(new Date(now));
-                return _tick=new Tick(seconds,s);
-            }
-            return _tick;
+            String s = ZonedDateTime.ofInstant(Instant.now(),_zoneId).format(_tzFormat);
+            return _tick=new Tick(seconds,s);
         }
+        return _tick;
```

Comment:
return the `tick` not the `_tick`.  Note that it does not matter if `_tick` changes after the local field is assigned as it means this call was just before the tick change and it is still valid to use the last tick as the results of the call can always be delayed into the next tick and beyond.